crnlib/crn_find_files.cpp

```diff
@@ -206,22 +206,39 @@ bool find_files::find_internal(const char* pBasepath, const char* pRelpath, cons
     if ((strcmp(ep->d_name, ".") == 0) || (strcmp(ep->d_name, "..") == 0))
       continue;
 
-    bool is_directory = 0;
-    bool is_file = 0;
+    bool is_directory = false;
+    bool is_file = false;
+    bool guessed = false;

```

Comment:
I would call it `known`. The word "guess" implies you are making the answer and don't know it it's correct.

Reply:
OK Done.